tests/fixtures/ask-about-output.ts

```diff
@@ -5,7 +5,7 @@ export const askAboutOutput = async (
   question: string,
 ): Promise<boolean> => {
   const completion = await anthropic.messages.create({
-    model: "claude-3-haiku-20240307",
+    model: "claude-3-5-sonnet-20240620",
```

Comment:
Ideally we use a cheaper model for the output questions becuse theyre easier, we could change this to the October Haiku version which is super smart and 1/3 the cost of sonnet, not a huge deal (but it does speed up runtimes)